pkg/apis/noobaa/v1alpha1/noobaa_types.go

```diff
@@ -223,6 +223,13 @@ type NooBaaSpec struct {
 	// DenyHTTP (optional) if given will deny access to the NooBaa S3 service using HTTP (only HTTPS)
 	// +optional
 	DenyHTTP bool `json:"denyHTTP,omitempty"`
+
+	// BucketLogging sets the configuration for bucket logging.
```

Comment:
let's keep this a one-liner. extra info will come from the properties' comments.

Reply:
Updated, Thanks.